src/app/qgisapp.cpp

```diff
@@ -703,6 +703,13 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
   mSplash->showMessage( tr( "Setting up the GUI" ), Qt::AlignHCenter | Qt::AlignBottom );
   qApp->processEvents();
 
+  QgsApplication::initQgis();
```

Comment:
you moved this part just in the middle of profile management... is there be better and clean place where to move the qgisInit() ? or does startup page depends on profile settings?